### DISCUSSION TITLE
x86 APK rejected because of 64-bit requirement, can be dropped?

### DISCUSSION THREAD

**Discussion by rdefalco** [airsdk/Adobe-Runtime-Support#638] [Q&A]
Hello,
for a new project Google forces me to comply to the 64-bit requirement. Using AIR33 I can package two APKs for ARM 32 and 64, and it gets accepted.

Once I try to add the x86 version APK, again Play Console rejects the app because it also wants the 64-bit counterpart.

I will drop x86 support for this project, but it's really a dead platform? Or are there plans to include x86_64 platform in future AIR releases?

Best regards,
Raffaele De Falco

**Comment by ajwfrost** [comment]
Hi

We're actually working on this one; there are some efforts under way to build AIR for Android x86_64 and add all the necessary toolchain updates. We'll need to do a fair amount of testing once the build is ready though so it's not going to be out for a few weeks I think (unless you want a beta version?)

thanks

**Comment by rdefalco** [comment]
Hello Andrew,
it's not a major issue, probably not an issue at all because (at least in Italy) I am not aware of recent smartphones using intel architecture, so we will continue to push x86 APK for our existing apps, and ARM-only for newer ones.

Anyway, packaging and publishing 4 APKs per app could be a bit frustrating ;-) probably we will wait for (eventually) App Bundle support, if feasible. Until then, ARM only except for our core apps.

Best regards
Raffaele
Falcosoft snc

**Comment by Oldes** [comment]
@rdefalco isn't x86(_64) version supposed to be used in Chrome OS?

**Comment by rdefalco** [comment]
Actually, I don't know, our target audience (sales agents and business owners' devices) are smartphone and tablets, sometimes Windows PCs. In Italy there is pratically no market for ChromeOS notebooks so we never had requests about that.
